api/db.py

```diff
@@ -26,6 +26,13 @@ class Database:
         Regression: 'regression',
     }
 
+    operator_map = {
```

Comment:
The universal convention for "constant" variables is to use capitalised names.  Not a real blocker, but see `COLLECTIONS` and not `collections` etc.  So keeping this consistent would probably be a good idea.